### PR TITLE
Implement DCF notices

### PR DESCRIPTION
**Pull request by jsturek**

### CODE REVIEW

**Comment by jsturek** [comment]
@skoolbus39 - We probably should update the DCF theme to style all contents of notice message to be white.  It currently does not style `<h(*)>` heading tags correctly.  I added inline style as a workaround for now.  Not sure if other elements will have the same issue if used in notice message.

File: www/js/submission.js [hunk 1]
```diff
@@ -373,7 +374,7 @@ define([
 						$('#deleteImages input[name=storyid]').val(data);
 					},
 					function (data, status, e) {
-						$('#dcf-main').prepend('<script type="text/javascript">WDN.initializePlugin("notice");</script><div class="wdn_notice negate"><div class="close"><a href="#" title="Close this notice">Close this notice</a></div><div class="message"><h4>Error</h4><p>Problem uploading image</p></div></div>');
+						$('#dcf-main').prepend('<script type="text/javascript">WDN.initializePlugin("notice");</script><div class="dcf-notice dcf-notice-danger" hidden><h2>Error</h2><div><p>Problem uploading image</p></div></div>');
```
Comment (jsturek) [inline www/js/submission.js:377]: Not sure if ever gets fired, so was not able to test.

File: www/templates/html/ENews/Newsroom/Stories.tpl.php [hunk 2]
```diff
@@ -14,9 +14,7 @@
 $cacheBust = uniqid();
 $savvy->loadScript(UNL_ENews_Controller::getURL() . "/js/manager.js?ver=" . $cacheBust);
 $savvy->loadScriptDeclaration("
-    WDN.loadJS(\"/wdn/templates_3.1/scripts/plugins/ui/jQuery.ui.js\");
-    WDN.loadCSS(\"/wdn/templates_3.1/css/content/forms.css\");
```
Comment (jsturek) [inline www/templates/html/ENews/Newsroom/Stories.tpl.php:18]: Appears to not be needed so removed

File: www/templates/html/ENews/Newsroom/Stories.tpl.php [hunk 3]
```diff
@@ -14,9 +14,7 @@
...
-
+    WDN.loadJS(\"/wdn/templates_5.3/js/compressed/plugins/ui/jquery-ui.js\");
```
Comment (jsturek) [inline www/templates/html/ENews/Newsroom/Stories.tpl.php:17]: Updated to 5.3, but not sure if needed.

**Review by ericras — APPROVED** [review]

**Pull request merged**